src/store/chats/normalization.ts

```diff
@@ -58,7 +63,14 @@ export const chatNormalizationSchema = new schema.Entity<INormalizedChat>(
         messages: chat.lastMessage
           ? {
               messages: {
-                [chat?.lastMessage?.id]: chat?.lastMessage as unknown as INormalizedMessage,
+                [chat?.lastMessage?.id]: {
```

Comment:
you don't need question marks here as you already check is `chat.lastMessage` exists